dds/DCPS/DomainParticipantImpl.cpp

```diff
@@ -650,9 +647,16 @@ DomainParticipantImpl::delete_topic_i(
           }
           return DDS::RETCODE_ERROR;
 
-        } else
+        } else {
           return DDS::RETCODE_OK;
-
+        }
+      } else {
+        if (DCPS_debug_level >= 1) {
```

Comment:
Should this be logged at a higher level?  I guess I'm not familiar with the issue to understand why someone would be interested in this log message.

Reply:
We could do that, which level, 4?

Reply:
I've never been quite sure what [it means by "administrative interfaces"](https://github.com/objectcomputing/OpenDDS/blob/c3e21a83403b91ea06e532506932bc9f8339722e/dds/DCPS/debug.h#L28).

Reply:
My guess would be things that are in between Entity creation/deletion and per-sample.  So other Entity "controls" like set_qos?